README.md

```diff
@@ -78,103 +101,366 @@ Modify the app's **android/app/src/main/AndroidManifest.xml** to include the fol
+method.
+```typescript
+export enum RealUserMonitoringTimings {
+  ViewLoaded = 'p',
+  NetworkCall = 'n'
+}
+```
+
+<br/>
+<br/>
+  
+###  User 
+The Raygun `User` type gets used to record session data, and maintain information for Real User Monitoring.
+This object type gets used as a parameter for the [setUser](#setuseruser-user--string) method.
+```typescript
+export type User = {
+  identifier: string;
+  isAnonymous?: boolean;
+  email?: string;
+  firstName?: string;
+  fullName?: string;
+  uuid?: string;
+};
+```
+
+<br/>
+<br/>
+
 ---
 
-# API Guide
+## Instantiating the client
 
-## init(RaygunClientOptions)
-The `init` method is used BEFORE doing anything else with the RaygunClient. This is important, as 
-all other functionality within the client will rely on the options parsed to it with this method.
+An instance of the RaygunClient is created by importing the client from the package.
+
+```typescript
+import RaygunClient from "raygun4reactnative"
+```
+This instance will be used to call action upon the rest of the API interface. Be sure to `init` your
+instance BEFORE using the other functions within the API.  
 
-#### Example Init:
+<br/>
+<br/>
+
+###  init(RaygunClientOptions) 
+The `init` function must be used BEFORE doing anything else with the RaygunClient. This is important, as 
+all other functionality within the client will rely on the options parsed to it with this function.
+
+Multithreading: This function, by default, is NOT asynchronous. BEWARE, as asynchronously calling 
+this method may lead to some undesirable outcomes. The `init` method needs to finish before all other
+methods.
 
 ```typescript
-RaygunClient.init({
-  apiKey: "This_Is_My_Key",
-  version: "0.1.2-beta",
-  enableCrashReporting: true,
-  disableNativeCrashReporting: false,
-  enableRealUserMonitoring: true,
-  disableNetworkMonitoring: false,
-  customCrashReportingEndpoint: "https://myCrashReportingEndpoint.com",
-  customRealUserMonitoringEndpoint: "https://myRealUserMonitoringEndpoint.com",
-  onBeforeSendingCrashReport: (crashReport) => console.log(crashReport),
-  ignoredURLs: ["http://thisIsAnInternalCall.com", "http://thisIsAnotherInternalCall.com"]
-})
+import RaygunClient, {RaygunClientOptions} from "raygun4reactnative"
+
+const options: RaygunClientOptions = {
+    apiKey: "This_Is_My_Key", 
+    version: "0.1.2-beta",
+    enableCrashReporting: true,
+    disableNativeCrashReporting: false,
+    enableRealUserMonitoring: true,
+    disableNetworkMonitoring: false,
+    customCrashReportingEndpoint: "https://myCrashReportingEndpoint.com",
+    customRealUserMonitoringEndpoint: "https://myRealUserMonitoringEndpoint.com",
+    onBeforeSendingCrashReport: (crashReport) => console.log(crashReport),
+    ignoredURLs: ["http://thisIsAnInternalCall.com", "http://thisIsAnotherInternalCall.com"]
+};
+
+// HERE
```

Comment:
What is this comment for?

Reply:
It was meant to be an indication of where the `init` example is. However, it's all relevant, so the comment is gone now.

DONE